### PR TITLE
Conference Search & Edit - initial

### PR DESCRIPTION
**Pull request by JamesSCBD**
1. Conference Filter & List - partial
2. Conference Add/Edit - partial

### CODE REVIEW

File: api/kronos.js [hunk 1]
```diff
@@ -145,6 +145,37 @@ export default class {
...
   }
 
+  // ====================
+  // Get Conference by id
+  // ====================
+  async getConference(id) {
+    // TODO - link api
+    // const data = await this.http.get(`api/v2018/conferences/${encodeURIComponent(id)}`).json();
+    const data       = await this.queryConferences(null);
+    const conference = data.records.find((c) => c.conferenceId === id);
```
Comment (sbilodeau) [inline api/kronos.js:155]: get should work

https://github.com/scbd/kronos/blob/master/Kronos.Server/Controllers/ConferencesController.cs#L52

File: pages/conferences/_conferenceId/index.vue [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+    <BForm ref="conferenceForm">
+      <PreventNavigation
+        :prevent="isDirty"
+        :actions="actions"
+        @save="saveOnLeave"
+        @next="$event.next()"
+        @cancel="$event.next(false)"
+      >
+        Do you want to Save?
+      </PreventNavigation>
```
Comment (sbilodeau) [inline pages/conferences/_conferenceId/index.vue:12]: this is cool no?

**Review by sbilodeau — CHANGES_REQUESTED** [review]

File: api/kronos.js (hunk 1 above)
Comment (JamesSCBD) [reply api/kronos.js:155]: Yes it is working...

File: pages/conferences/_conferenceId/index.vue (hunk 2 above)
Comment (JamesSCBD) [reply pages/conferences/_conferenceId/index.vue:12]: Yupp

**Pull request merged**